src/rez/pip.py

```diff
@@ -335,6 +335,7 @@ def pip_install_package(source_name, pip_version=None, python_version=None,
     """
     installed_variants = []
     skipped_variants = []
+    variant_types = []
```

Comment:
I can't find any references to this variable in the function. I think it's no more needed.